sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -6387,129 +6403,1760 @@
+                    "conditional": {
+                      "show": null,
+                      "when": null,
+                      "eq": "",
+                      "json": ""
+                    },
+                    "customConditional": "",
+                    "logic": [],
+                    "attributes": {},
+                    "overlay": {
+                      "style": "",
+                      "page": "",
+                      "left": "",
+                      "top": "",
+                      "width": "",
+                      "height": ""
+                    },
+                    "type": "htmlelement",
+                    "input": false,
+                    "tableView": false,
+                    "placeholder": "",
+                    "prefix": "",
+                    "suffix": "",
+                    "multiple": false,
+                    "defaultValue": null,
+                    "protected": false,
+                    "unique": false,
+                    "persistent": false,
+                    "clearOnHide": true,
+                    "refreshOn": "",
+                    "redrawOn": "",
+                    "dataGridLabel": false,
+                    "labelPosition": "top",
+                    "description": "",
+                    "errorLabel": "",
+                    "tooltip": "",
+                    "hideLabel": false,
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "dbIndex": false,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "widget": null,
+                    "validateOn": "change",
+                    "validate": {
+                      "required": false,
+                      "custom": "",
+                      "customPrivate": false,
+                      "strictDateValidation": false,
+                      "multiple": false,
+                      "unique": false
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "id": "evqdh87"
+                  }
+                ]
+              },
+              {
+                "components": [
+                  {
+                    "label": "CSLFTotal",
+                    "tag": "p",
+                    "className": "",
+                    "attrs": [
+                      {
+                        "attr": "",
+                        "value": ""
+                      }
+                    ],
+                    "content": "{{(data.disbursement?.disbursement1cslf ? data.disbursement.disbursement1cslf : 0) + (data.disbursement?.disbursement2cslf ? data.disbursement.disbursement2cslf : 0)}}",
```

Comment:
The expression can be simplified as below
````
{{(data.disbursement?.disbursement1cslf ?? 0) + (data.disbursement?.disbursement2cslf ?? 0)}}
````

Reply:
avoiding ?? operator for the benefit that it is not supported in 2 browsers.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator